packages/editor-ui/src/components/NodeSettings.vue

```diff
@@ -158,6 +185,21 @@ export default mixins(
 
 				return this.nodeType.properties;
 			},
+			scopesContent (): string {
+				return this.$locale.baseText(
+					'nodeSettings.scopes',
+					{
+						adjustToNumber: this.scopes.length,
+						interpolate: {
+							activeCredential: this.activeCredential,
+							scopes: this.scopesToDisplay,
+						},
+					},
+				);
+			},
+			scopesToDisplay(): string {
+				return this.scopes.map(scope => scope.replace(/\//g, '/<wbr>')).join('<br>');
```

Comment:
what's `<wbr>`?

Reply:
hmm interesting

Reply:
why does this need to be removed?

Reply:
Not removing, I'm adding a word break opportunity after every slash so the URL wraps cleanly.